src/js/views/patients/worklist/worklist-list.scss

```diff
@@ -27,6 +27,7 @@
   color: #999;
   display: inline-block;
   vertical-align: middle;
+  width: 16px;
```

Comment:
For the alignment to be correct, the details icon`<div>` needs to have a `width: 16px`. Whether the details icon `<svg>` is being displayed or not.

We removed this from the details icon in PR: https://github.com/RoundingWell/care-ops-frontend/pull/856.

Specifically via this code change: https://github.com/RoundingWell/care-ops-frontend/pull/856/files#r1026919485.